crates/core_arch/src/lib.rs

```diff
@@ -40,7 +40,10 @@
     decl_macro
 )]
 #![cfg_attr(test, feature(test, abi_vectorcall))]
-#![cfg_attr(all(test, target_arch = "wasm32"), feature(wasm_simd))]
+#![cfg_attr(
+    all(test, any(target_arch = "wasm32", target_arch = "wasm64")),
+    feature(wasm_simd)
+)]
```

Comment:
In C/C++, we have macros `__wasm32__`, `__wasm64__`, and `__wasm__`, for things specific to 32-bit wasm, things specific to 64-bit wasm, and things common to both, respectively. Would it make sense for Rust to define a `wasm` predicate to help users writing code that works on both wasm32 and wasm64?

Reply:
That seems like a good idea to me, I'm not immediately sure how to go about doing that though.

Reply:
Alright added `cfg(wasm)` to the rustc pr :+1: